src/frontend/src/session.rs

```diff
@@ -615,10 +615,47 @@ impl Session for SessionImpl {
             )));
         }
         let stmt = stmts.swap_remove(0);
-        let rsp = infer(self, stmt, sql).map_err(|e| {
-            tracing::error!("failed to handle sql:\n{}:\n{}", sql, e);
-            e
-        })?;
+        let rsp = match stmt {
+            Statement::Query(_) => infer(self, stmt, sql).map_err(|e| {
+                tracing::error!("failed to handle sql:\n{}:\n{}", sql, e);
+                e
+            })?,
+            Statement::ShowObjects(show_object) => match show_object {
+                ShowObject::Columns { table: _ } => {
+                    vec![
+                        PgFieldDescriptor::new("Name".to_owned(), TypeOid::Varchar),
+                        PgFieldDescriptor::new("Type".to_owned(), TypeOid::Varchar),
+                    ]
+                }
+                _ => {
+                    vec![PgFieldDescriptor::new("Name".to_owned(), TypeOid::Varchar)]
+                }
+            },
+            Statement::ShowVariable { variable } => {
+                let name = &variable[0].value.to_lowercase();
+                if name.eq_ignore_ascii_case("ALL") {
+                    vec![
+                        PgFieldDescriptor::new("name".to_string(), TypeOid::Varchar),
+                        PgFieldDescriptor::new("setting".to_string(), TypeOid::Varchar),
+                        PgFieldDescriptor::new("description".to_string(), TypeOid::Varchar),
```

Comment:
```suggestion
                        PgFieldDescriptor::new("Name".to_string(), TypeOid::Varchar),
                        PgFieldDescriptor::new("Setting".to_string(), TypeOid::Varchar),
                        PgFieldDescriptor::new("Description".to_string(), TypeOid::Varchar),
```

Reply:
I refer from https://github.com/singularity-data/risingwave/blob/2719006246c213f765d5ecc14596fa4979994de8/src/frontend/src/handler/variable.rs#L86.
If we want to unify, we need to unify in all place?